pkgs/servers/teleport/default.nix

```diff
@@ -0,0 +1,50 @@
+# This file was generated by https://github.com/kamilchm/go2nix v2.0-dev
+{ stdenv, buildGoPackage, zip, fetchurl, fetchgit, fetchhg, fetchbzr, fetchsvn }:
+
+buildGoPackage rec {
+  name = "teleport-unstable-${version}";
+  version = "2.4.0";
+
+  goPackagePath = "github.com/gravitational/teleport";
+  subPackages = [ "tool/tctl" "tool/teleport" "tool/tsh" ];
+  buildInputs = [ zip ];
+  postBuild = ''
+    pushd .
+    cd $NIX_BUILD_TOP/go/src/github.com/gravitational/teleport
+    mkdir -p build
+    echo "making webassets"
+    make build/webassets.zip
+    cat build/webassets.zip >> $NIX_BUILD_TOP/go/bin/teleport
+    rm -fr build/webassets.zip
+    cd $NIX_BUILD_TOP/go/bin
+    zip -q -A teleport
+    popd
+    '';
+
+  dontStrip = true;
+
+  # This repo has a private submodule "e" which fetchgit cannot handle
+  # without failing. Using fetchurl instead on the latest tagged release.
+  src = fetchurl {
+    url = "https://github.com/gravitational/teleport/archive/v2.4.0.tar.gz";
+    sha256 = "01qcr6vml3ias3gbkqjnb5fmc6ap1fd5mi52ygn3agafy63jb5rd";
+  };
+
+  /*
+  src = fetchgit {
+    rev = "df6caeae7d90e520478244d583cd9e60d924fb32";
+    url = "https://github.com/gravitational/teleport";
+    sha256 = "05f0a0z831rdn77ji5p6xd27cpyj2c2jncivaby5qkqj1w0kdpn5";
+  };
+  */
+
+  goDeps = ./deps.nix;
```

Comment:
Try to ignore `goDeps` here, there's missing vendor scaning logic in go2nix v2. Packages from `Gopkg.lock` should be included only when they doesn't exists in `vendor` dir.

Reply:
I confirm this works well without `deps.nix`.
I didn't realize `goBuildPackage` uses the `vendor` directory and I don't really know what to think about this: I like to be able to know what are the build requires by querying the Nix store and this is no longer possible when the `vendor` directorys used. But this is another story:)

For now, I think we could remove `deps.nix`.

Reply:
removing....